site/index.html

```diff
@@ -109,76 +193,120 @@
 				v-38.99c0-30.188,12.875-59.094,35.313-79.292l64.292-57.865c4.5-4.042,7.063-9.802,7.063-15.854s-2.563-11.813-7.063-15.854
-				l-64.292-57.865C140.875,162.083,128,133.177,128,102.99V64h256v38.99C384,133.177,371.125,162.083,348.688,182.281z"/>
-			<path d="M329.521,149.333H182.469c-4.219,0-8.042,2.49-9.75,6.344c-1.708,3.865-0.99,8.365,1.854,11.49l74.271,68.521
+				l-64.292-57.865C140.875,162.083,128,133.177,128,102.99V64h256v38.99C384,133.177,371.125,162.083,348.688,182.281z" />
+                                    <path d="M329.521,149.333H182.469c-4.219,0-8.042,2.49-9.75,6.344c-1.708,3.865-0.99,8.365,1.854,11.49l74.271,68.521
 				c2.031,1.844,4.594,2.76,7.156,2.76s5.125-0.917,7.156-2.76l74.26-68.521c2.844-3.125,3.563-7.625,1.854-11.49
-				C337.563,151.823,333.74,149.333,329.521,149.333z"/>
-		</g>
-	</g>
-</g>
-</svg>
-            Tempo estimado: <b>02h00m</b></span>
-            <div class="contacts">
-                <p>Para mais informações ou dúvidas sobre a operação a decorrer utilize o e-mail <a href="mailto:ajuda@nau.edu.pt">ajuda@nau.edu.pt</a></p>
+				C337.563,151.823,333.74,149.333,329.521,149.333z" />
+                                </g>
+                            </g>
+                        </g>
+                    </svg>
+                    <span id="maintenance-esttime">Tempo estimado</span>: <b><time
+                            datetime="PT2H00M">02h00m</time></b></span>
+                <div class="contacts">
+                    <p><span id="maintenance-message">Para mais informações ou dúvidas sobre a operação a decorrer
+                            utilize o serviço de apoio ao utilizador </span> <a
+                            href="https://ajuda.nau.edu.pt">https://ajuda.nau.edu.pt</a></p>
+                </div>
             </div>
-        </div>
-    </section>
-    <p class="email">
-        E-mail: <a href="mailto:ajuda@nau.edu.pt">ajuda@nau.edu.pt</a>
-    </p>
-    <section class="social">
-        <p>
-            <a class="facebook-link" href="https://www.facebook.com/plataformanau/" target="_blank">Facebook</a>
-        </p>
-        <p>
-            <a class="twitter-link" href="https://twitter.com/nau_plataforma" target="_blank">Twitter</a>
-        </p>
-    </section>
+        </section>
+        <p class="website"><a href="https://ajuda.nau.edu.pt">https://ajuda.nau.edu.pt</a></p>
+        <section class="social">
+            <ul>
+                <li>
+                    <a class="facebook-link" href="https://www.facebook.com/plataformanau/" target="_blank">Facebook</a>
+                </li>
+                <li>
+                    <a class="twitter-link" href="https://twitter.com/nau_plataforma" target="_blank">Twitter</a>
+                </li>
+            </ul>
+            <button id="changeLang" class="change-lang-link" aria-label="Mudar língua para Português">Português</button>

```

Comment:
I think will be best if he language changer would be the first content, aligned to the right.